core/src/main/java/org/apache/carbondata/core/scan/filter/FilterUtil.java

```diff
@@ -180,9 +185,27 @@ private static FilterExecuter createFilterExecuterTree(
    * @return
    */
   private static FilterExecuter getIncludeFilterExecuter(
-      DimColumnResolvedFilterInfo dimColResolvedFilterInfo, SegmentProperties segmentProperties) {
-
-    if (dimColResolvedFilterInfo.getDimension().isColumnar()) {
+      DimColumnResolvedFilterInfo dimColResolvedFilterInfo,
+      MeasureColumnResolvedFilterInfo msrColResolvedFilterInfo,
+      SegmentProperties segmentProperties) {
+    if (null != msrColResolvedFilterInfo && msrColResolvedFilterInfo.getMeasure().isColumnar()) {
```

Comment:
I don't think `msrColResolvedFilterInfo.getMeasure().isColumnar()` is really required. it is only for dimensions.please remove